openedx/features/wikimedia_features/wikimedia_general/api/v0/urls.py

```diff
@@ -0,0 +1,19 @@
+"""
+Urls for Messenger v0 API(s)
+"""
+from django.conf import settings
+from django.conf.urls import url
+
+from openedx.features.wikimedia_features.wikimedia_general.api.v0.views import (
+    GetCourseFont
+)
+
+app_name = 'general_api_v0'
+
+urlpatterns = [
+    url(
+        fr'^course_font/{settings.COURSE_KEY_PATTERN}',
```

Comment:
Instead of course_font can we change this to wiki_metadata? This helps in sending additional custom strings  in future.